driver/compile_common.ml

```diff
@@ -0,0 +1,118 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           *)
+(*                                                                        *)
+(*   Copyright 1996 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+open Misc
+open Compenv
+
+type info = {
+  sourcefile : string;
+  modulename : string;
+  outputprefix : string;
+  env : Env.t;
+  ppf_dump : Format.formatter;
+  tool_name : string;
+}
+
+
+let cmx i = i.outputprefix ^ ".cmx"
+let obj i = i.outputprefix ^ Config.ext_obj
+let cmo i = i.outputprefix ^ ".cmo"
+let annot i = i.outputprefix ^ ".annot"
+
+let init ppf_dump ~init_path ~tool_name ~sourcefile ~outputprefix =
```

Comment:
~init_path is a strange name. Shouldn't this be called ~native?

Reply:
Well, it's the boolean that control of path should be initialized (which is apparently needed for native). I wouldn't object renaming it.